quic/s2n-quic-events/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "s2n-quic-events"
 # this in an unpublished internal crate so the version should not be changed
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
this is an internal crate; we use this crate to generate our events.

Reply:
fyi: https://github.com/aws/s2n-quic/blob/main/scripts/events is the script you can run to generate events. it reads some rust files and generates [rust code](https://github.com/aws/s2n-quic/blob/main/quic/s2n-quic-core/src/event/generated.rs). This is desired because there are a couple things we want per event (trait impl, annotations, testing) and its easier to do this in an automated fashion.